lib/compat/wordpress-6.5/scripts-modules.php

```diff
@@ -17,6 +17,8 @@
 	 *
 	 * @since 6.5.0
 	 *
+	 * @global WP_Script_Modules $wp_script_modules.
```

Comment:
Let's make it the same as the text in the core.

https://github.com/WordPress/wordpress-develop/blob/0acfd3a72c66c7544b3ef9e183ee1af3958ae3c3/src/wp-includes/script-modules.php#L17

Reply:
> Let's make it the same as the text in the core.
> 
> https://github.com/WordPress/wordpress-develop/blob/0acfd3a72c66c7544b3ef9e183ee1af3958ae3c3/src/wp-includes/script-modules.php#L17

@t-hamano Thanks for the feedback. 

I have addressed changes suggested changes